### PR TITLE
Mm 26

### PR DESCRIPTION
**Pull request by AnastasiyaPoleshuk**
ticket: https://magicmark.atlassian.net/jira/software/projects/MM/boards/1?selectedIssue=MM-26

### CODE REVIEW

File: login/login.js [hunk 1]
```diff
@@ -1,40 +1,68 @@
...
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const moment = require('moment');
+const db = require('../queries/queries');
 
-const loginUser = (req, res) => {
-  if (!req.body) return res.sendStatus(400);
+const loginUser = async (req, res) => {
+  if (!req.body) {
+    return res.sendStatus(400);
```
Comment (AyvengoBY) [inline login/login.js]: Could you please use Http Stattus codes package here ?

File: user/user.js [hunk 2]
```diff
@@ -1,47 +1,79 @@
...
 const StatusCodes = require('http-status-codes');
+const db = require('../queries/queries');
 
-const getUser = (req, res) => {
-  const token = req;
-  const responseData = getUserData(token);
-  res.status(responseData.status).send(responseData.data);
+const getUser = async (req, res) => {
+  if (!req) {
+    return res.sendStatus(400);
```
Comment (AyvengoBY) [inline user/user.js]: Please, use HttpStatuses here

File: login/login.js (hunk 1 above)
Comment (AnastasiyaPoleshuk) [reply login/login.js]: fixed

File: user/user.js (hunk 2 above)
Comment (AnastasiyaPoleshuk) [reply user/user.js]: fixed

**Review by AyvengoBY — APPROVED** [review]

**Pull request merged**